files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -156,6 +156,10 @@
   cert, key = OmnibusHelper.gen_certificate
 end
 
+# TODO: we are now using public_keys instead of certs and the
+#       bootstrap process now uses the public key. This is still
+#       here for backwards compatibility in case it is used anywhere
+#       else but we should consider deprecating it in the future.
```

Comment:
No problem from the ec-metal side,  but we should make this change SUPER clear in the release notes, for our customers that bootstrap their Chef servers in whacky ways  (cc @ryancragun )